_posts/en/newsletters/2019-02-19-newsletter.md

```diff
@@ -0,0 +1,250 @@
+  needs to take towards finalizing it.
+
+- [Bitcoin Core #14075][] adds a `keypool` parameter to the
+  `importmulti` RPC that allows imported public keys to be added to the
+  keypool---the list of keys that are used to create new receiving and
+  change addresses.  The option is only available for wallets that have
+  private keys disabled (see [PR#9662][Bitcoin Core #9662] described in
+  [Newsletter #5][]).  This allows a user of a cold wallet or a hardware
+  wallet to import their public keys into a watching-only Bitcoin Core
+  wallet and then receive payments normally.  When attempting to spend
+  payments, the wallet can generate an unsigned transaction---including
+  a change address---using a [BIP174][] PSBT and send that to a tool
+  such as [HWI][] that will connect to the external wallet for review
+  and signing.
+
+- [Bitcoin Core #14021][] changes the `importmulti` RPC to store any key
+  origin metadata included as part of an [output script
+  descriptor][descriptor].  The [key origin information][] specifies what HD
+  seed and derivation path was used to generate a public key.  When key
+  origin metadata is available in the wallet, any PSBTs generated by the
+  wallet will include that data to allow hardware wallets or other
+  programs to locate the private keys needed to sign the PSBT.
+  See the footnote[^fn-example] for an example of key origin information
+  in a descriptor.
+
+- [Bitcoin Core #14481][] updates the `listunspent` RPC to include a
+  `witnessScript` field for payments sent to either a native P2WSH
+  address or a P2SH-nested P2WSH address.  This information is sourced
+  from the wallet (as it's not actually part of the unspent output) and
+  so it will only be available for solvable addresses (addresses whose
+  inputs the wallet knows how to include in an unsigned transaction).
+
+- [Bitcoin Core #15063][] allows the wallet to fallback on [BIP21][] parsing
+  of a `bitcoin:` URI if [BIP70][] support has been disabled.  As
+  specified by [BIP72][], the `bitcoin:` URI was extended in a
+  backwards-compatible way to contain an additional `r=` parameter
+  containing the BIP70 URL.  This was done to allow services already
+  using BIP21 URIs to upgrade to supporting BIP70 without losing
+  existing users.  However, now that many wallets are services are
+  deprecating their BIP70 support, the same mechanism can be used in
+  reverse so that services that previously supported BIP70 can allow
+  their non-BIP70 users to continue to get payment details by just
+  clicking on a `bitcoin:` link.
+
+- [Bitcoin Core #15153][] adds a GUI menu to open a wallet and
+  [#15195][Bitcoin Core #15195] adds a menu to close a wallet.  This
+  makes it much easier to use Bitcoin Core's multiwallet mode from the
+  GUI, although it's not yet possible to create a wallet from the GUI
+  without using the debug console (that todo is the final item on the
+  [dynamic wallet checklist][Bitcoin Core #13059]).
```

Comment:
Order mixed up here?

Reply:
Is your comment that the parenthetical seems to imply that the checklist desires the ability to create a wallet from the debug console (rather than, as intended, that it desires a menu item)?  I can fix that, but I want to ensure that's the problem you're describing.

Reply:
Sorry, I thought it should be `[Bitcoin Core #13059][dynamic wallet checklist]` instead of `[dynamic wallet checklist][Bitcoin Core #13059]`. But that was wrong :-)